Content/Console/paket.dependencies

```diff
@@ -1,6 +1,5 @@
 source https://www.nuget.org/api/v2
 source https://api.nuget.org/v3/index.json
-framework: netcoreapp2.1
```

Comment:
Why not update this to match the TFM for the console app?

Reply:
I ask because removing this adds a ton of lines to the lockfile to cover TFMs that aren't going to be used.

Reply:
I'm fine either way honestly.  It just stinks having this TFM in multiple places.